DebugUtils/BUILD

```diff
@@ -0,0 +1,21 @@
+load("@rules_cc//cc:defs.bzl", "objc_library")
+
+exports_files([
```

Comment:
To make sure things work with Swift, rather than exporting files like this, make them `objc_library`'s that have a `hdrs` to expose the header.

Reply:
Sounds good